src/fastcs_odin/util.py

```diff
@@ -162,3 +162,47 @@ def _walk_sub_controllers(
     for sub_controller in controller.get_sub_controllers().values():
         yield sub_controller
         yield from _walk_sub_controllers(sub_controller)
+
+
+def unpack_status_arrays(parameter: list, uri: list[list[str]]):
+    """Takes a list of OdinParameters and a list of special uri. Search the parameter
+    for elements that match the values in the uri list and split them into one
+    new odinParameter for each value.
+
+    Args:
+        parameter: List of OdinParameters
+        uri: List of special uris to search and replace
+
+    Returns:
+        list[OdinParameters]
```

Comment:
```suggestion
    Returns: Original list of parameters with status arrays replaced with indexed parameters
```

Reply:
The type is given by the function signature. What we want is a description of what is returned.